lib/membrane_rtc_engine/media_event.ex

```diff
@@ -61,6 +61,44 @@ defmodule Membrane.RTC.Engine.MediaEvent do
     |> do_create(:broadcast)
   end
 
+  @spec create_peer_tracks_removed_event(peer_id_t(), map()) ::
+          sfu_media_event_t()
+  def create_peer_tracks_removed_event(peer_id, track_ids) do
+    %{
+      type: "peerTracksRemoved",
```

Comment:
```suggestion
      type: "tracksRemoved",
 ```     
 
 and `create_tracks_removed_event`